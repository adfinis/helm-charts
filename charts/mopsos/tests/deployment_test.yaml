suite: deployment tests
templates:
  - deployment.yaml
tests:
  - it: should match snapshot
    asserts:
      - hasDocuments:
          count: 1
      - matchSnapshot: {}
