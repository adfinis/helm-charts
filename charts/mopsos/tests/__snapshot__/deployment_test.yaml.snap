should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: mopsos
        app.kubernetes.io/version: v0.3.4
        helm.sh/chart: mopsos-0.5.3
      name: RELEASE-NAME-mopsos
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: mopsos
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: mopsos
        spec:
          containers:
          - envFrom:
            - secretRef:
                name: RELEASE-NAME-mopsos-secret
            - configMapRef:
                name: RELEASE-NAME-mopsos-config
            image: ghcr.io/adfinis-sygroup/mopsos:v0.3.4
            imagePullPolicy: IfNotPresent
            livenessProbe:
              httpGet:
                path: /health
                port: http
            name: mopsos
            ports:
            - containerPort: 8080
              name: http
              protocol: TCP
            readinessProbe:
              httpGet:
                path: /health
                port: http
            resources: {}
            securityContext: {}
          securityContext: {}
          serviceAccountName: RELEASE-NAME-mopsos
